### PR TITLE
fix: correct instances of camel-case-uri to camel-case-uris

### PR DESCRIPTION
**Pull request by nikklassen**

### CODE REVIEW

**Review by GregFurmanek — COMMENTED** [review]
There appear to be a number of instances of this mismatch, including in tests. Perhaps we should correct and test all of them? tagging: @noahdietz

**Comment by noahdietz** [comment]
@nikklassen would you mind fixing the other broken documentation examples in this same PR? You can say "no" :)

File: lint/rule_urls.go [hunk 1]
```diff
@@ -8,13 +8,14 @@ import "strings"
 // At Google, we inject additional rule URL mappings into this list.
 // Example: google_rule_url_mappings.go
 // package lint
-// func init() {
-//   ruleURLMappings = append(ruleURLMappings, internalRuleURLMapping)
-// }
 //
```
Comment (GregFurmanek) [inline lint/rule_urls.go:11]: Is this change intentional?

Comment (nikklassen) [reply lint/rule_urls.go]: I reverted the change, it's due to go 1.19's `gofmt`

**Review by GregFurmanek — APPROVED** [review]
